rudder-agent/SOURCES/rudder-agent-postinst

```diff
@@ -73,6 +73,8 @@ if [ ! -f /var/rudder/cfengine-community/inputs/promises.cf ] || ! /var/rudder/c
 then
   rm -rf /var/rudder/cfengine-community/inputs/* || true
   cp -r /opt/rudder/share/initial-promises/* /var/rudder/cfengine-community/inputs/
+  rm -rf /var/rudder/ncf/common/*
+  cp -r /opt/rudder/share/initial-ncf/* /var/rudder/ncf/common/
 fi
```

Comment:
I think this whole block of code should live in check-rudder-agent (or soon to be "rudder agent check"). The command is already called at the end of this script (the last thing that happens) and it should be doing this reset anyway.

However, that would likely involve changing the order below so that the agent start command comes after the check.